### PR TITLE
Add CheckAppProtocolAnnotation to check-gke-ingress

### PR DESCRIPTION
**Pull request by songrx1997**
/assign @swetharepakula

### CODE REVIEW

File: cmd/check-gke-ingress/app/ingress/rule_test.go [hunk 1]
```diff
@@ -350,3 +350,70 @@ func TestCheckIngressRule(t *testing.T) {
 		}
 	}
 }
+
+func TestCheckAppProtocolAnnotation(t *testing.T) {
```
Comment (swetharepakula) [inline cmd/check-gke-ingress/app/ingress/rule_test.go:354]: nit: add test cases with malformed annotations as well.

Comment (songrx1997) [reply cmd/check-gke-ingress/app/ingress/rule_test.go:354]: Updated

**Review by swetharepakula — COMMENTED** [review]
/lgtm
/approve

**Review by swetharepakula — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**